### PR TITLE
Add QR code to presentation

### PR DESCRIPTION
**Pull request by avan1235**

### CODE REVIEW

File: docs/final_presentation.html [hunk 1]
```diff
@@ -445,7 +445,10 @@
 class: center, middle
 
 <h2 style="margin-top: 0; margin-bottom: 0.5em;">Thank you!</h2>
-https://github.com/prinz-nussknacker/prinz
+
+[https://github.com/prinz-nussknacker/prinz](https://github.com/prinz-nussknacker/prinz)
```
Comment (kantoniak) [inline docs/final_presentation.html]: `remark.js` links urls by default.

File: docs/images/qr.svg [hunk 2]
```diff
@@ -0,0 +1,686 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (kantoniak) [inline docs/images/qr.svg]: If you have time, could you run this code through [SVGOMG](https://jakearchibald.github.io/svgomg/)? This simplifies file a lot.

Comment (kantoniak) [reply docs/images/qr.svg]: There's also Google Charts APIs: https://chart.googleapis.com/chart?chs=300x300&cht=qr&chl=https://github.com/prinz-nussknacker/prinz

**Review by kantoniak — APPROVED** [review]

File: docs/final_presentation.html (hunk 1 above)
Comment (avan1235) [reply docs/final_presentation.html]: thanks for pointing, will fix

File: docs/images/qr.svg (hunk 2 above)
Comment (avan1235) [reply docs/images/qr.svg]: looks better, will change

**Pull request merged**